pro-planner-client/src/redux/userSlice.js

```diff
@@ -0,0 +1,33 @@
+import {createSlice} from '@reduxjs/toolkit';
+
+const initialState = {
+    userList: [
+        {
```

Comment:
Looking at the "Redux/Data" outline that we have, this list should mainly just be an array of strings, not one of objects. To build a bit more intuition for that, adding a user will send a call to the backend to setup the logic for a new user. The backend should handle making the information and establishing the source of truth for which users exist for the plan, so I don't believe the frontend should manage that or we would have a possible disconnect in information.

This is more so just a note that we may have to refactor this part of the logic out when we start integration with our backend.